product_docs/docs/migration_portal/4/03_mp_using_portal/mp_ai_copilot/ai_good_prompts.mdx

```diff
@@ -3,21 +3,21 @@ title: How to create good prompts
 
 ---
 
-The quality and accuracy of the answers the AI Copilot provides depends highly on your prompt. 
+The quality and accuracy of the answers that the AI Copilot provides depend on the quality of your prompt. 
```

Comment:
Hey @mlewandowski-edb! 
Our editor, @ebgitelman, made a good point about how we sometimes say "the AI Copilot" and sometimes "AI Copilot." I know the guideline for "Migration Portal" is to skip the article. Should we do the same here for "AI Copilot"?

Reply:
@ebgitelman It is capitalized in the UI, so let's go for AI Copilot for now?

Reply:
AI Copilot is fine, without the "the." I'll make those changes in this PR.

Reply:
Thank you! 🙏